src/main/java/org/apache/brooklyn/cloudfoundry/entity/VanillaCloudfoundryApplication.java

```diff
@@ -77,14 +82,38 @@
     AttributeSensor<String> ROOT_URL =
             Sensors.newStringSensor("webapp.url", "URL of the application");
 
-    AttributeSensor<CloudFoundryPaasLocation> CLOUDFOUNDRY_LOCATION = Sensors.newSensor(
-            CloudFoundryPaasLocation.class, "cloudFoundryWebApp.paasLocation",
-            "CloudFoundry location used to deploy the application");
-
     AttributeSensor<Boolean> SERVICE_PROCESS_IS_RUNNING = Sensors.newBooleanSensor(
             "service.process.isRunning",
             "Whether the process for the service is confirmed as running");
 
     AttributeSensor<Lifecycle> SERVICE_STATE_ACTUAL = Attributes.SERVICE_STATE_ACTUAL;
 
+    AttributeSensor<Map> APPLICATION_ENV = Sensors.builder(Map.class, "application.env").build();
```

Comment:
`ENV` is a `ConfigKey` and `APPLICATION_ENV` is a sensor that shows the current env of an application. For example, if you add an env using the `setEnv` effector,  `APPLICATION_ENV` will show the new added env